### DISCUSSION TITLE
Introductions: Say hi!

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/AI-Chat-App-Hack#5] [General]
Welcome to the AI Chat App Hack. Please use this thread to say hi!

Suggested details:

* 📛 name/nickname/username
* 🌎 location (country and state/province if relevant)
* 🤖 previous AI or python experience
* 😻 fav hobby outside coding

**Comment by pamelafox** [comment]
I'll start...


📛 Pamela Fox (pamela, not pam!)
🌎 Bay Area, California
🤖 I've been Python'ing since ~2005, my college years, and dove into LLMs in last year. When I was in college, AI was Prolog ;-)
😻 I spend most of my 'free' time parenting 2 young kids but I sneak in some gardening of pollinator-friendly plants occasionally.

**Reply by ghost** [reply]
Hi, loved the intro😂. What's a LLMS , if you don't mind me asking?

**Reply by pamelafox** [reply]
LLMs = Large Language Models - like GPT-3.5, GPT-4, etc.

**Reply by multispark** [reply]
LLM stands for Large Language Model! These are AI models trained on large sets of data and can be specialized for different purposes. Looks like Pamela beat me to the punch!

**Reply by GeraltOfCodea** [reply]
Your session at the AI Tour in San Francisco was amazing! I look forward to participating in this Hackathon.

**Reply by gu12934** [reply]
📛 Gurm
🌎 Canada,AB
🤖 2-3 years python experience
😻 playing pokemon and working out

**Reply by CharanBharathula** [reply]
Hi everyone. GenAI was really a good flavor of the season. I was excited to have a good connections to build good projects and ideas
Charan
India
Python and AI enthusiastic
Developing AI and android apps and reading books, surfing youtube

**Comment by multispark** [comment]
Nice to meet everyone, can't wait to see all the cool projects that are submitted!

📛 Josh Oh
🌎 Seattle, Washington
🤖 I used Python a lot in university, and still do now. More recently focusing on LLMs!
😻 I enjoy playing guitar, saxophone, and singing.

**Reply by TIKIALINE** [reply]
من الجيد مقابلة الجميع ، لديا قابلية للتعلم و لرؤية جميع المشاريع الرائعة!

📛 محمد تكيالين
🌎 الشلف، الجزائر
🤖 لم استخدم بايثون بعد  ، و ارغب في  أن أفعل ذلك الآن. في الآونة الأخيرة بدأت التعرف  على  LLMs!
😻 أنا أستمتع بتدريب حاملي المشاريع الصغيرة والمتوسطة ، امتلك خبرة تزيد عن العشرين سنة  التنشيط والمرافقة .

**Comment by ghost** [comment]
📛 Eugene Stephen.
🌍  Eastlands, Nairobi
🤖. I have Intermediate level experience in Python started using Python last year. Major doing projects using Python.
   😻I spend my free time  doing  photography and filming.

**Reply by treezy254** [reply]
yow, ninja... si tupige colab

**Comment by ghost** [comment]
Thanks ,

**Comment by FlyingPhish** [comment]
📛 Jan
🌎 England
🤖 Scripting and programming for 7+ years, Python for over a year. Recently jumped on the Rust train. Primarily a heavy user of ChatGPT and have since release. I've recently starting to invest more time in LLMs.
😻 Cyber security, gaming and getting out on the motorbike when I can.

**Comment by ThunderRoar** [comment]
Hello there!

📛 Shreyas
🌎 Ontario, Canada
🤖 Worked with Python, Java, Javascript, and C. Recently started exploring LLMs and machine learning models 
😻 I like to bike, read books and play sports such as badminton and volleyball.

**Comment by Pratikdate** [comment]
Hii everyone, Can't wait to build project. 
📛  Pratik Date
🌍 Pune ,India
🤖 Worked in python ,Java , JS, Dart . experience in CNN but new in LLMs.
😻 I Like to read books and build tech for society.

**Comment by NitkarshChourasia** [comment]
I am thrilled to join this fantastic community!

📛 Nitkarsh Chourasia
🌎 Mumbai, Maharashtra, India
🤖 I love working with Python Programming Language. I maintain a repository with over 12k+ forks and 30k+ Stars dedicated to various scripts and projects made in Python Programming Language.
😻 Being with family, friends and strolling in garden.

**Comment by AntareepDey** [comment]
Hey Everyone! 
📛 Antareep Dey
🌎 India 
🤖 Programming in python for last 4 years, built an basic chatbot that helps with medical Q&A using LLM 's last semester as a part of a university project. Still lots to figure out !
😻 I enjoy reading and photography in my  free time.

**Reply by chillywilly7** [reply]
Hi! What were some of the major challenges you faced during your project?

**Comment by GeraltOfCodea** [comment]
Hello! I am excited to be here and participate. This is my first hackathon!

📛 Ryan Norrbom - GeraltOfCodea on GitHub
🌎 Tillamook, Oregon - USA
🤖 I started in Python several years ago. I have a RAG chatbot built on Databricks w/Python and Azure Function apps and am working on building one on Azure AI.
😻 I spend a lot of time coding these days, I am in a Software Developer BA program and work as an AI Architect for a Creamery on the Oregon coast. I enjoy reading books on AI and Quantum Computing. Most of all though, I enjoy Camping/Vacationing with my wife and young kids!

**Reply by pallavigadepalli** [reply]
Ryan would love to connect on linkedin and brainstorm since you are on the PST TZ. linkedin.com/in/pallavigadepalli

**Comment by Nikhil8557** [comment]
Jai siya ram, I'm a student and Microsoft ambassador from India. Let's create a better future ✨️

**Comment by giftahmed** [comment]
Hello, 
I am Gift Ahmed from Nigeria and a student . Nice to meet you all

**Comment by NitkarshChourasia** [comment]
Welcome.

**Comment by VardhanReddy2** [comment]
Hey all, I am really excited joining this community. It would be my first hackathon.

📛 Vishnu Vardhan Reddy Ireddy
🌎 Silicon Valley, Bay Area, California
🤖 My experience with Python began in my undergraduate years around 2017, and I've been exploring AI and machine learning topics during my master's program. I'm particularly interested in applying these skills in innovative software engineering projects.
😻 Outside of coding, I enjoy hitting the gym and maintaining an active lifestyle. It's a great way to balance the demands of tech studies and work.

**Comment by komalvenugopal** [comment]
Hello Everyone 👋

📛 Komal Venugopal Vattumilli
🌎 San Jose, California, USA
🤖 I have a strong background in software engineering, with 2.5 years of experience as a DevOps Engineer. Currently pursuing a Master's in Software Engineering with a specialization in Cloud Computing. While I have a solid foundation in Python, I am relatively new to LLMs but eager to dive deep into this fascinating field.
😻 When I'm not coding or buried in cloud technologies, I love exploring the outdoors, hiking, and experimenting with photography.

Looking forward to learning from all of you and collaborating on some innovative projects!

Warm regards,
Komal

**Comment by NitkarshChourasia** [comment]
Welcome to the community.
I want a pair programming partner.
Any body intrested for it?

**Reply by tajbender** [reply]
What kind of application or service do you want to create?

**Comment by amjeed-ay** [comment]
📛 Abdulmajid Yunusa
🌎Bauchi, Nigeria
🤖Software / Game Dev, New to AI 
😻 listening to music

**Comment by Thukyd** [comment]
Hey, 
📛 Jan-Philipp
🌎 Living in Barcelona but I am German
🤖 In the field of Conversational Ai since 2017, I worked with Python / Node.js
😻 Learning new things, history, cycling, meeting with friends in the various restaurants and tapas bars in the city...

**Reply by pallavigadepalli** [reply]
Hello nice to connect with you Jan-Philip would love to brainstorm after the hackathon.

**Reply by Thukyd** [reply]
Sure, always open for that!

**Reply by tajbender** [reply]
Hi, I am German too - 🇩🇪 

Feel free to contact me

**Comment by pallavigadepalli** [comment]
📛 Pallavi Gadepalli
🌎 United States and India
🤖  I don't have any python experience but my team member in India does and we are building it together. Building something in conversational AI.
😻 fav hobby outside coding - singing - semi professional, hiking, pelotoning, canyoning

**Reply by NitkarshChourasia** [reply]
I have also been very much interested in LLM's as my final year project on which I am also working and thinking to output an IEEE format research paper. 
https://github.com/NitkarshChourasia/alive_document

**Reply by FlyingPhish** [reply]
Looks like solid work! When will you be releasing the source?

**Reply by NitkarshChourasia** [reply]
Around May.

**Comment by DiegoLopezInc** [comment]
**Excited to learn during this Hackathon!** 😊
Thanks @pamelafox for making me aware of this on youtube shorts! 

📛 Diego
🌎 North Carolina, United States
🤖 Some bioinformatics course work
😻 Meeting new people, reading books, learning

**Comment by ArpitaisAn0maly** [comment]
📛 Arpita
🌎 CA, USA
🤖 Coding python since 2016
😻 music, fitness, travelling

**Comment by mattgotteiner** [comment]
📛 Matt Gotteiner
🌎 Seattle, WA
🤖 I really dove into python this last year and have learned a lot! My AI journey really kicked off when ChatGPT took off and I haven't looked back. 
😻 Spending time with my dog

**Comment by zedhaque** [comment]
📛 zedhaque
🌎 Sydney, Australia
🤖 I have hacked a AI chatbot using the code [Azure-Samples/azure-search-openai-demo](https://github.com/Azure-Samples/azure-search-openai-demo). Maybe I am not allowed to participate? I would however love to join and learn new tricks. I am a python newbie.
😻 Eating and Travelling

**Reply by NitkarshChourasia** [reply]
Let's connect.

**Reply by multispark** [reply]
Feel free to still participate!

**Reply by zedhaque** [reply]
Thanks. I will build something new :) now that the repo supports free tier!

**Comment by christopher-mierbach** [comment]
📛 Chris
🌎 Cyprus
🤖 Total Newbie in IT, but working on it 😄 Certification as a "Microsoft certified AI Engineer" and first lines in python few weeks ago. 
😻 Sports, eating and time with my wonderful family (🙋‍♀️ & 6 months old 👶)

**Comment by codebytes** [comment]
📛 Chris Ayers
🌎 Tampa, Florida, USA
🤖 I've been coding since second grade, I picked up python in the late 90s, dabbled a bit and moved on to .NET. I circled back to python in the last few years with the rise of AI and Data Science.
😻 I spend a lot of free time speaking, blogging (https://chris-ayers.com), gaming, and nerding out on stuff.

**Comment by navyajammalamadaka** [comment]
📛 Navya Jammalamadaka(Jam)
🌎 Bay Area, California
🤖 Coding in Python since 2017
😻 I've been really pickup up a passion for baking recently.

**Comment by khelanmodi** [comment]
📛 Khelan Modi
🌎 Seattle, WA
🤖 I've been Python-ing since my college days in 2021, and I jumped into LLMs just last year. Always been into AI stuff, playing around with OpenAI from way back. Can't wait to share with y'all on building RAG apps with just a database! 🚀
😻 Snowboarding, sports, fitness, traveling, reading books

**Comment by srenna** [comment]
📛 Sabrina Renna 
🌎 Toronto, ON 
🤖 Python since 2021 | Presently completing a Masters of Management in AI @ Schulich. 
😻 Coffee, canines and cooking.

Let's connect!
[https://www.linkedin.com/notifications/?filter=all](url)

**Comment by KomaliValluru** [comment]
📛 Komali Valluru
🌎 Tampa, FL, USA
🤖 I am a CS student from University of South Florida. Started on LLM's last month. 
😻 Love playing with my cat.

**Comment by crissins** [comment]
📛 Cristopher Hugo Olivares Del Real/Cris/crissins
🌎 Ciudad Juarez Chihuahua
🤖 AI 900 certified 
😻 Playing Guitar and spending time with my family

**Comment by anudeepk224** [comment]
Hey all, I am really excited joining this community. It would be my first hackathon.

📛 Anudeep Goud Kotagiri
🌎 Silicon Valley, Bay Area, California
🤖 My experience with Python began in my undergraduate years around 2018, and I've been exploring AI and machine learning topics during my master's program. I'm particularly interested in applying these skills in innovative software engineering projects.
😻 Outside of coding, I enjoy hitting the gym and maintaining an active lifestyle. It's a great way to balance the demands of tech studies and work.

**Comment by ai-by-humans** [comment]
Hey all have fun on the project.
📛Mark
🌎Netherlands
🤖working on a few closed projects with the goal to "humanize ai"  
😻Trying to go to every festival in the netherlands

**Comment by ssccinng** [comment]
hi
📛 scixing
🌎 china
🤖 robotllm,
😻 pokemon

**Reply by ssccinng** [reply]
For some reasons, it seems that I don't have enough time to complete this project, but I am still happy to participate in learning. Please make up for the following items in the future

**Comment by tajbender** [comment]
nice to meet you 😀 

 📛 Scheunenproduktion.de
 🌎 Germany
 🤖 tajbender
 😻 Barney

**Comment by Sethmaurice** [comment]
📛 Seth
🌎 Rwanda
🤖 Machine learning (python & django)
😻 Football

**Comment by vdekercd** [comment]
Hello all

I'm interested in joining a team!
If you are looking for a member, contact me :)

📛 damienvdk
🌎 Belgium
🤖 I've been interested in creating chatbots for a few years now. I've learned about the Microsoft Bot Framework and the DialogFlow service. Recently I've been looking at the pytorch library and I'd like to go further. 
😻 I like playing rackets sports like ping pong and tennis. I'm also an avid Clash of clan player.

**Reply by NitkarshChourasia** [reply]
@pallavigadepalli

**Reply by tajbender** [reply]
@tajbender

**Comment by Sshankar7** [comment]
Hello everyone! Finally found something before the start date. ;)

📛 Siddharth Shankar
🌎 India
🤖 I've been working on Python since 7+ years building tools, automating stuffs and improving processes. Also, I've worked primarily on Machine Learning in last 4 years. 
😻 Playing computer games, watching movies in theatres; currently playing Avatar: Frontiers of Pandora

**Comment by keshavsingh3197** [comment]
Hi, nice to meet everyone,

📛 Keshav Singh/keshav/keshavsingh4522
🌎 Jaipur Rajasthan India
🤖 I am familiar with this and have done some more stuffs at the college time at that time python was my first choice for coding.
😻 Listening the music, playing cricket etc.

**Comment by mscherotter** [comment]
Hi everyone!  I'm Michael - a 16-year Microsoft veteran who is now independent starting up [Galeryst](https://galeryst.com) - a site that procedurally generates 3D virtual art galleries and  jumping into Generative AI for the creative fields!

**Comment by treezy254** [comment]
Hello everyone, 

📛 I'm Samuel Kamau
🌎 Nairobi, Kenya
🤖 Hacker and developer.
😻 Videogames, party, music.

**Reply by juanfmuller** [reply]
Favorite video game? 

Skyrim take this one for me haha

**Comment by heftif** [comment]
📛 Fiona
🌎 Switzerland
🤖 Have a solid base in Python and I'm interested in learning how we can use LLMs with our data to create a great chatbot.
😻 Gaming, building miniature models, sports

**Comment by ferrarirosso** [comment]
📛 Nello
🌎 Switzerland
🤖 Very little Python experience, developer since 30 years (maybe more...)
😻 Family, cooking, coding and my cool dog

**Comment by chukxman** [comment]
Chukwudi Nnaji from Nigeria 
I am here to learn and explore

**Comment by Narsimha96** [comment]
📛 Narsimha
🌎 Chicago,United states.
🤖 4+ years Python experience, developer since 6 years
😻 Family, cooking, coding, snooker and books

**Comment by JoyeetaBais** [comment]
>Hiii!
> 
> 
> * 📛 Joyeeta Bais
> * 🌎 India
> * 🤖 I am totally begineer in this field, I know python but never used for development
> * 😻 i like designing and dancingg

**Comment by guifreitag** [comment]
📛 Guilherme
🌎 São Paulo, Brazil
🤖 I know little python, only from courses and little experiments.
😻 Right now busy raising 2 beautiful daughters :-)

**Comment by juanfmuller** [comment]
📛 Juan Muller here!
🌎 I’m currently in Texas but will be relocating to NYC in March. Excited for the move.
🤖 I’ve built some projects using the OpenAI API for GPT models. I have about 2 years experience in Python mostly for DSA but also in Django and FastAPI. For work I use Swift TypeScript and Java, however.
😻 Hanging out with friends is definitely my favorite thing to do on my time off! You can usually find me watching TV or playing video games when I’m free on the weekdays.

**Comment by chillywilly7** [comment]
Hello Everyone, 

📛Daniel Gutierrez
🌎 I am in Austin, Texas
🤖 My first Python pip install was in 2019 ~ approx. five years ago while in grad school.  I have 2 years solid experience for my current role.  I have been using Python primarily to build automation scripts, dashboards, web crawlers, but also have gotten my feet wet with FastAPI, Selenium, and Django. I am relatively new to LLMs but super excited to learn and be able to connect the dots. 
😻 On my free time I enjoy cooking, watching TV shows, eating out in new spots around town and enjoying nature

**Comment by cjrieth** [comment]
Hi everyone!

📛 CJ Rieth
🌎 Bay Area, California
🤖 I am a recent grad from UW and have been using python since high school, so probably close to 6 years now! I don't have much formal experience with LLMs aside from some guest lectures I attended, but I am looking forward to learning more about them through this hackathon.
😻 Outside of coding I enjoy cooking, hiking, playing music, and hanging out with friends!

**Comment by JasonHaley** [comment]
Hello everyone!

Jason Haley (some socials @haleyjason)

Hamilton, Massachusetts (north shore of Boston)

Just started digging into AI last November and this month have been focusing on RAG - that seems to be the most practical way my day job will use AI in the short term (I'm and Independed Consultant with mostly small business customers).

Outside of coding -> last year I started ultra running but also started roasting my own coffee at home. When I'm not running, I'm probably with one of my pets (english cream golden retriever or black cat - indoor only).

**Comment by shiv2-dot** [comment]
Hi Everyone,
I'm Shivarajesh, also known as @bluelion0007.
India
Python and AI enthusiast
Have good exposure to Python and AI
Worked on several projects and currently doing self-research on AI/ML

**Comment by mimoflynn** [comment]
Hi there, good to be in this hackathon / community :) 

📛 Mim
🌎 Dublin, Ireland
🤖 Python 'improver' (coding locally, but lots to learn re deployment/DevOps!), good exposure to AI and LLMs 
😻 Fave hobby:  "The Arts" especially live music gigs, theatre & cinema. Went to an Andy Warhol exhibition last week - stunning!

**Comment by jeremykpark** [comment]
Hello,

📛 Jeremy K
🌎 Toronto, Canada
🤖 have been dabbling for a few years with AI at home. Long time web+php+js programmer too. Also picked up Unity and C# in the past 3 years
😻 writing music, building computers / studios, and exploring the world

**Comment by kevball2** [comment]
📛 Kevin
🌎 North Carolina, USA
🤖 Programming with Python casually for several years. In my day job I am an Azure DevOps Engineer.
😻 Gaming, Biking and automating all the things!

**Reply by pallavigadepalli** [reply]
Hi Kevin! I will reachout if I have an issue. thanks for introducing yourself.

**Reply by pallavigadepalli** [reply]
@kevball2  any chance you can help me out. I am not even able to get past getting to the repo. If you can I would love some assistance for 10 minutes to get started I can go forward from then on. thank you!

**Comment by Tusharbhosalehub** [comment]
Hie...
📛 I'm Tushar Bhosale
🌎 Pune, India
🤖 CS Engineering Student
😻 Sports, Gaming, Trekking, Coding...

**Comment by bob17dz** [comment]
Hello, I'm Bakar
📛 Boubakeur/Bakar/b1b7
🌎 Algeria
🤖 Experiences { PC user +15y/html css c# sketchUp AI +1y/ } 
😻 learning,drinking coffee,

**Comment by IsaacGemal** [comment]
Hey there! I'm Isaac
📛 Isaac is fine
🌎 From New Jersey
🤖 I'm a civil engineer so I don't have much formal python experience, just scripts here and there for work/home
😻 Big fan of photography

**Comment by zzulueta** [comment]
Hi there, good to be in this hackathon / community :)

📛 Ziggy
🌎 Makati, Philippines
🤖 newbie in Python. Mostly use C#.
😻 Fave hobby: Going to the gym. Playing games when there is time. Watching Netflix with my gf.

**Comment by ksut** [comment]
Hello,
📛 Kalendra
🌎 Texas, USA
🤖 Using Python in Data Engineering. I played with Open AI API but had no deep experience in AI. Have 15+ years of experience in Data Analytics 
😻 Reading, walking/running, Yoga.

**Comment by MatheusHRV** [comment]
Hi, is a pleasure to participate with you!!

📛 Matheus
🌎 São Paulo, Brazil
🤖 I've worked almost 3 years with Python for data analysis and web scraping, also built some chatbots with LLMs.
😻 I like arts in all kinds of forms, even my linkedin profile photo is a parody of the incredible 'Girl with a Pearl Earring' painting, i'm also a poetry passionate.

**Comment by maxquinteros** [comment]
Hello everyone, this is the second hackathon in which I will participate with my team.

📛 Maximiliano Quinteros
🌎 Antofagasta, Chile
🤖 I studied python and AI while I was a college student.I don't have much experience but I am excited to participate and learn.
😻 I like sports, the gym and cooking

**Comment by 01vadim10** [comment]
📛 Vadzim Zylevich
🌎 Copenhagen, Denmark
🤖 I am new to Python, but I have 10 years of experience with C#. I want to try to implement a project using C#
😻 I like going to the gym, reading books, writing articles, and watching TV shows

**Comment by daisy-amrita** [comment]
Hey there:)

Amrita Singh
India
I have experience working with CNN (Convolutional Neural Networks) for tasks involving classification and regression. However, I'm relatively new to LLMs (Large Language Models).
Looking forward to learning more about LLMs through this hackathon.

Warm Regards
Amrita

**Comment by Diego0106** [comment]
Hello there!

📛 Diego0106
🌎 Ecuador
🤖 I have experience programming personal projects about artificial intelligence.
😻 I really like listening to music and watching series
